NEWS

```diff
@@ -64,16 +39,32 @@ Major new features:
 - Use UCX allocator for OSHMEM symmetric heap allocations to optimize intra-node
   data transfers.  UCX SPML only.
 - Use UCX multi-threaded API in the UCX PML.  Requires UCX 1.0 or later.
+- Added support for Flux PMI
+- Update embedded PMIx to version 2.0.0
+- Update embedded hwloc to version 1.11.3
 
 Changes in behavior compared to prior versions:
 
+- Per Open MPI's versioning scheme (see the README), increasing the
+  major version number to 3 indicates that this version is not
+  ABI-compatible with prior versions of Open MPI.  You will need to
+  recompile MPI and OpenSHMEM applications to work with this version
+  of Open MPI.
+- With this release, Open MPI supports MPI_THREAD_MULTIPLE by default.
+- New configure option for specifying location of libnl.
+- New configure option for specifying location of zlib.
+- New configure option to request Flux PMI support.
+- The help menu for mpirun and related commands is now context based.
+  "mpirun --help compatibility" generates the help menu in the same format
+  as previous releases.
+
 Removed legacy support:
-- "yoda" SPML component is removed from the Open SHMEM codebase. It was utilizing
+- AIX is no longer supported.
+- Loadlever is no longer supported.
+- "yoda" SPML component is removed from the Open SHMEM code base. It was utilizing
   Open MPI BTL components for supported networks. Please see README for the currently
   available options.
```

Comment:
Minor amendment - "OpenSHMEM currently supports the UCX and MXM transports via the `ucx` and `ikrit` SPMLs respectively."